DesignSafe-CI/portal
# Bug/DES-2038 - Rename and Colorized Tabs

## Overview: ##
The CSS classes used to describe the colored tabs in the projects area are vague. I've renamed them to describe the colors they represent and fixed a few typos in the templates which were showing the wrong colors for certain entity tabs.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2038](https://jira.tacc.utexas.edu/browse/DES-2038)

## Summary of Changes: ##

## Testing Steps: ##
1. Visit the publication preview areas and published projects to make sure the colors are displaying as outlined in the `publication-preview.scss` file.

## UI Photos:
BEFORE
![before](https://user-images.githubusercontent.com/29575979/151032169-ab6e3d82-10d6-41da-948c-ac5b3bdceae5.png)
AFTER
![after](https://user-images.githubusercontent.com/29575979/151032198-0a1861a6-94c0-4a8a-8415-74cab5476b53.png)

## Notes: ##
